### DISCUSSION TITLE
kustomize-controller: Add HelmRelease to Kustomization health check

### DISCUSSION THREAD

**Discussion by phillebaba** [fluxcd/flux2#187] [Proposals] [answered]
Currently the Kustomization controller supports health checks for Deployment, DaemonSet and StatefulSet resources. It would be nice to add HelmRelease to the list of supported resources as they will be added be added by the kustomization controller.

In theory it would be possible to add a health check for the Deployment that is created by the Helm chart, but i think it would be cleaner to not do that.

**Comment by stefanprodan** [comment, marked as answer]
Yes we should extend the health check to support the HelmRelease kind. There is an issue opened for this here https://github.com/fluxcd/kustomize-controller/issues/68

**Reply by phillebaba** [reply]
Ah I missed that issue. As I know I will need this, I can implement the feature if nobody has started working on it yet.
